src/test/java/com/hotels/beeju/core/BeejuCoreTest.java

```diff
@@ -29,17 +29,49 @@
 import org.apache.hadoop.hive.conf.HiveConf;
 import org.apache.hadoop.hive.metastore.HiveMetaStoreClient;
 import org.apache.hadoop.hive.metastore.api.Database;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 public class BeejuCoreTest {
+  
+  private String preKey = "my.custom.pre.key"; 
+  private String preValue = "my.custom.pre.value";
+  private String postKey = "my.custom.pre.key"; 
+  private String postValue = "my.custom.pre.value"; 
```

Comment:
should these say `my.custom.post.key` and `my.custom.post.value` ?

Reply:
Yes they should, well spotted. will update now.